### PR TITLE
[not to merge] dev notes

### PR DESCRIPTION
**Pull request by bastilian**

### CODE REVIEW

File: src/PresentationalComponents/Common/Tables.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { generateFilter } from '@redhat-cloud-services/frontend-components-utilities/helpers';
 import { SYSTEM_TYPES } from '../../AppConstants';
 
+// All this feels like it would be better in a helpers file or something.
```
Comment (adonispuente) [inline src/PresentationalComponents/Common/Tables.js:4]: Most of the functions are all helpers used inside different tables (Pathways,Rules,Systems tables) and inventory component. Others Like workloadQueryBuilder are used in 11 files. In functions that are used across more than just table components, what if we put those in a 'AppHelpers file' and then keep the functions relative to just tables here, but rename this file 'TableHelpers

File: src/PresentationalComponents/Export/SystemsPdfBuild.js [hunk 2]
```diff
@@ -102,6 +103,7 @@ export const tablePage = ({ page, systems, intl }) => {
   ];
 
   return (
+    // that fragment is not needed at all/
```
Comment (adonispuente) [inline src/PresentationalComponents/Export/SystemsPdfBuild.js:106]: When working on these I found this feature isnt working. The network request fails on select, need to look into further

File: src/cveToRuleid.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// What is this for?
```
Comment (adonispuente) [inline src/cveToRuleid.js:1]: Seems that in Detail.js under Recs Directory that and Item is mapped againsted this to see if the item is actually a CVE. Im not sure exactly what a CVE is but it looks like if whatever item is indeed a cve, a request is sent to vulnerability api

File: src/Utilities/BarDividedList.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import './_BarDividedList.scss';
 
+// Not used anywhere.
```
Comment (adonispuente) [inline src/Utilities/BarDividedList.js:4]: addressed

Comment (adonispuente) [inline src/Utilities/BarDividedList.js:4]: addressed

File: src/PresentationalComponents/Cards/Pathways.js [hunk 5]
```diff
@@ -36,6 +36,7 @@ import messages from '../../Messages';
 import { strong } from '../../Utilities/intlHelper';
 import { useIntl } from 'react-intl';
 
+// There are like three components in here... separate files.
```
Comment (adonispuente) [inline src/PresentationalComponents/Cards/Pathways.js:39]: addressed

File: src/PresentationalComponents/Common/Common.js [hunk 6]
```diff
@@ -22,13 +22,15 @@ const cache = createIntlCache();
 const locale = navigator.language.slice(0, 2);
 const intl = createIntl(
   {
+    // Yo, no.
```
Comment (adonispuente) [inline src/PresentationalComponents/Common/Common.js:25]: addressed

File: src/PresentationalComponents/Labels/CategoryLabel.js [hunk 7]
```diff
@@ -12,18 +12,29 @@ import { useIntl } from 'react-intl';
...
     labelList.map((label) => label.id)
   );
 
+  // This should be a component.
   const label = (icon, text) => (
     <Label icon={icon} variant="outline" color="blue" isCompact={isCompact}>
       {text}
     </Label>
   );
+  // This is not the way.
```
Comment (adonispuente) [inline src/PresentationalComponents/Labels/CategoryLabel.js:29]: addressed

File: src/PresentationalComponents/Labels/CategoryLabel.js [hunk 8]
```diff
@@ -12,18 +12,29 @@ import { useIntl } from 'react-intl';
...
   const sortedFrequency = (arr) =>
+    // 🧐
     Object.entries(
       arr.reduce((acc, curr) => ((acc[curr] = (acc[curr] || 0) + 1), acc), {})
     ).sort((a, b) => b[1] - a[1]);
   const sortedFrequencyList = sortedFrequency(
     labelList.map((label) => label.id)
   );
 
+  // This should be a component.
```
Comment (adonispuente) [inline src/PresentationalComponents/Labels/CategoryLabel.js:23]: addressed

File: src/PresentationalComponents/Labels/RecommendationLevel.js [hunk 9]
```diff
@@ -8,10 +8,13 @@ const RecommendationLevel = (props) => {
   const intl = useIntl();
   const { recommendation_level: lvl, isCompact } = props;
 
+  // Another component that is not a component
```
Comment (adonispuente) [inline src/PresentationalComponents/Labels/RecommendationLevel.js:11]: addressed

File: src/PresentationalComponents/Loading/Failed.js [hunk 10]
```diff
@@ -6,6 +6,7 @@ import React from 'react';
 import propTypes from 'prop-types';
 
 const Failed = ({ message }) => (
+  // Not needed fragment
```
Comment (adonispuente) [inline src/PresentationalComponents/Loading/Failed.js:9]: addressed

File: src/PresentationalComponents/Loading/Loading.js [hunk 11]
```diff
@@ -3,6 +3,7 @@ import { CardBody } from '@patternfly/react-core/dist/js/components/Card/CardBod
 import { List } from 'react-content-loader';
 import React from 'react';
 const Loading = () => (
+  // not needed fragment.
```
Comment (adonispuente) [inline src/PresentationalComponents/Loading/Loading.js:6]: addressed

File: src/PresentationalComponents/Modals/DisableRule.js [hunk 12]
```diff
@@ -31,7 +31,7 @@ const DisableRule = ({
     host !== undefined || hosts.length > 0
   );
 
-  // eslint-disable-next-line no-unused-vars
+  // eslint-disable-next-line no-unused-vars ... ? lol.
```
Comment (adonispuente) [inline src/PresentationalComponents/Modals/DisableRule.js:34]: addressed

File: src/SmartComponents/SystemAdvisor/index.js [hunk 13]
```diff
@@ -1 +1,2 @@
+// ?
```
Comment (adonispuente) [inline src/SmartComponents/SystemAdvisor/index.js:1]: This is vital to inventory loading atm, but I dont know why. Im not 100% sure how SystemAdvisor,Inventory, and InventoryDetail communicate

File: src/SmartComponents/SystemAdvisor/EmptyStates.js [hunk 14]
```diff
@@ -16,6 +16,7 @@ import {
 import MessageState from '../../PresentationalComponents/MessageState/MessageState';
 import PropTypes from 'prop-types';
 
+// These should be in separate files and in a components directory or presentational
```
Comment (adonispuente) [inline src/SmartComponents/SystemAdvisor/EmptyStates.js:19]: Addressed

File: src/PresentationalComponents/AppIntro/IntroBody.js [hunk 15]
```diff
@@ -61,6 +65,7 @@ const IntroBody = ({ isPreProduction }) => {
           {intl.formatMessage(messages.introTakeTour)}
```
Comment (adonispuente) [inline src/PresentationalComponents/AppIntro/IntroBody.js:65]: We dont even have an obj defined for this, im not sure this component is being used in Advisor

File: src/PresentationalComponents/Common/Tables.js (hunk 1 above)
Comment (adonispuente) [inline src/PresentationalComponents/Common/Tables.js:4]: This file is basically just helper functions used across the app. I might just rename to 'TableHelpers'

File: src/Store/AppReducer.js [hunk 16]
```diff
@@ -17,6 +18,7 @@ export function systemReducer(cols, INVENTORY_ACTION_TYPES) {
         })),
       };
     },
+    // This is elsewhere too. only use one.
```
Comment (adonispuente) [inline src/Store/AppReducer.js:21]: im not sure if this is outdated now, but I dont see where else this is used? Its 'defined' here and only used in inventory

**Comment by bastilian** [comment]
@adonispuente I'll close this it'll still remain here and we can pull it out for reference, but it'll be out of my PR list. :D Thank you for all your efforts on improving Advisor and more! <3

**Pull request closed without merging**